.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "mixer"]
-	path = mixer
-	url = https://github.com/datacommonsorg/mixer.git
```

Comment:
Is this change and the mixer submodule change intentional?  Please revert if not.